src/lab/CustomText/CustomText.tsx

```diff
@@ -0,0 +1,86 @@
+import { IconButton, TextField, useTheme } from '@material-ui/core';
+import EditIcon from '@material-ui/icons/Edit';
+import SaveIcon from '@material-ui/icons/Save';
+import React from 'react';
+import { CustomTextProps } from './base';
+import { useStyles } from './styles';
+
+const CustomText: React.FC<CustomTextProps> = ({
+  value,
+  onChangeText,
+  validateText,
+  helperText,
+  placeholder,
+  label,
+  showLabelOnDisabled = true,
+  showHelperTextOnDisabled = true,
+  width,
+  multiline = false,
+  initialStateOutline = false,
+  isEditable = true,
```

Comment:
We can extend `TextField` interface from MUI and add only those that are custom like validateText, helperText, etc.